aioxmpp/ibr/service.py

```diff
@@ -135,7 +134,7 @@ def get_client_info(self):
         A query is sent to the server to obtain the client's data stored at the
         server.
 
-        :return: :class:`xso.Query`
+        :return: :class:`~aioxmmp.ibr.Query`
```

Comment:
typo

Reply:
I don't see the typo. (Although the text should be reworded.)

Reply:
aioxmmp ≠ aioxmpp

Reply:
Ow, reading error correction is evil!